packages/v4/src/content/design-guidelines/components/menu/menu.md

```diff
@@ -0,0 +1,82 @@
+---
+id: Menu
+section: components
+---
+
+A menu component that can be used in a variety of contexts whenever the user needs to choose between multiple values, options, or actions. A menu can be opened in a [dropdown](/components/dropdown), a [select list](/components/select), or it can be revealed upon right clicking on a region within a page.
+
+## Elements and variations
+### Action menu
+An actions menu is used for presenting a list of actions or links.
+
+<img src="./img/basic-menu.png" alt="actions dropdown" width="863"/>
```

Comment:
I feel like the pink headers in this image should be in sentence case? I'm guessing you title cased it because it's referring to the bullets, but I think it looks odd since they're just general "statements"